mozilla_django_oidc/utils.py

```diff
@@ -159,3 +162,88 @@ def add_state_and_verifier_and_nonce_to_session(
         "nonce": nonce,
         "added_on": time.time(),
     }
+
+
+class AuthorizationCodeRequestMixin:
```

Comment:
Although it makes a lot of sense to introduce this mixin and use it in both places, I am concerned that introducing this change will also introduce complexity and it will make it harder to modify/debug/override code. Thoughts?

Reply:
@akatsoulas and I discussed the pros and cons of this via video. Closing this in favor of https://github.com/mozilla/mozilla-django-oidc/pull/515.